grammars.yml

```diff
@@ -1119,6 +1119,10 @@ vendor/grammars/vscode-lean:
 - markdown.lean.codeblock
 - source.lean
 - source.lean.markdown
+vendor/grammars/vscode-lean4:
+- markdown.lean4.codeblock
+- source.lean.markdown
```

Comment:
Do not manually modify this to solve the test failure as this file is managed by scripts. There fix needs to be made in the grammar. As you've stated, Lean != Lean 4 so the same scope names can't be used by two different languages' grammars.

Reply:
This is now fixed upstream, I've updated the submodule.